motion/p5js/regularity/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Marius Claßen
+author-url: https://github.com/mariusclassen
+title: spiral beams
+tags: [P5.js, motion, law of regularity]
```

Comment:
The "Law of Regularity" does not exist currently. If you want to introduce a new law you need to write a definition.